cnxarchive/cache.py

```diff
@@ -69,9 +71,10 @@ def search(query, query_type, nocache=False):
                 # search with one term or one filter, plus query_type
             cache_length = int(settings['search-long-cache-expiration'])
 
-        # store in memcache
-        mc.set(mc_search_key, search_results, time=cache_length,
-               min_compress_len=1024*1024)  # compress when > 1MB
+        if len(mc_search_key) <= mc.server_max_key_length:
+            # store in memcache
+            mc.set(mc_search_key, search_results, time=cache_length,
+                   min_compress_len=1024*1024)  # compress when > 1MB
```

Comment:
Is there still the chance that the `mc_search_key` is longer than the max length? I thought we were removing this kind of condition with the use of a hash?

Reply:
Oh there were 2 places that have this condition... I forgot to remove this one.  I'll update.